multi-soil-moisture/observation-types/soil-moisture/view.js

```diff
@@ -0,0 +1,16 @@
+function(endDevice, latestObservation) {
+  return {
+    observationsMapValue: 'moisture',
+    observationsMapGauge: 'moisture',
+    fields: {
+      temperature: {
+        value: latestObservation.data.moisture,
+        name: 'moisture',
+        text: latestObservation.data.moisture[4].toFixed(2) + 'VWC%',
+        color: latestObservation.data.moisture[4] > 80 ? 'danger' : 'success',
```

Comment:
Do you know this will have enough elements to satisfy the `[4]` indexer?

Reply:
No not really, and there's no array bounds check in JS so I'm taking advantage of that. :-)